worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -361,6 +366,7 @@ class FetchHandler(
     logDebug(s"Received req from ${NettyUtils.getRemoteAddress(client.getChannel)}" +
       s" to fetch block $streamChunkSlice")
 
+    incChunkAppConnectionCounter(streamChunkSlice.streamId)
```

Comment:
Shouldn't increase this counter here. This happened when client is fetching a block from server.